### PR TITLE
feat(Labels): Various UI/UX improvements

### PR DESCRIPTION
**Pull request by grafakus**
### ✨ Description

**Related issue(s):** resolves https://github.com/grafana/explore-profiles/issues/88 and more

A couple of UI/UX improvements for the "Labels" exploration type:

- The "Select" action has been removed on panels with a single label value (further breakdown does not make sense)
- The "Compare" action has been removed when there's only a single panel in the grid (nothing to compare to)
- Bar gauges for single label values are not helpful (see https://github.com/grafana/explore-profiles/issues/88), they have been replaced by Stats visualizations, e.g.: 
  image
- The "Bar gauge" button on the panel type switcher has been renamed to "Aggregate" (works for when the UI displays bar gauge or stat panels):
  image

### 📖 Summary of the changes

See diff tab for specific comments

### 🧪 How to test?

- See related issue + manual testing in local after checking out this PR's branch

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[bug] "
```
Comment (grafakus) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: Not related to this PR, auto-linting by the pre-commit hook

File: src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/SceneByVariableRepeaterGrid.tsx [hunk 2]
```diff
@@ -234,6 +236,14 @@ export class SceneByVariableRepeaterGrid extends SceneObjectBase<SceneByVariable
   subscribeToHideNoDataChange() {
     const noDataSwitcher = findSceneObjectByClass(this, SceneNoDataSwitcher) as SceneNoDataSwitcher;
 
+    if (!noDataSwitcher.isActive) {
```
Comment (grafakus) [inline src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/SceneByVariableRepeaterGrid.tsx:239]: Just a slight improvement

File: src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/SceneByVariableRepeaterGrid.tsx [hunk 3]
```diff
@@ -266,7 +276,7 @@ export class SceneByVariableRepeaterGrid extends SceneObjectBase<SceneByVariable
     };
   }
 
-  getCurrentOptions(variable: QueryVariable): VariableValueOption[] {
+  getCurrentOptions(variable: QueryVariable, panelTypeFromSwitcher: PanelType): VariableValueOption[] {
```
Comment (grafakus) [inline src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/SceneByVariableRepeaterGrid.tsx:279]: TODO: refactor to extract the "groupBy" logic out of this class.

**Review by simonswine — APPROVED** [review]
LGTM

I think the Totals works a lot better than aggregations, but there might even be a better options (I can't think of one).

Also when reviewing this I thought about this: https://github.com/grafana/explore-profiles/issues/95

**Pull request merged**